OSDOCS-12237-man-console: Bug batching 4.18 management console (#88893)
openshift/openshift-docs: release_notes/ocp-4-18-release-notes.adoc

* Previously, the *Sample* tab did not show when creating a new ConfigMap with one or more ConfigMap ConsoleYAMLSamples present on the cluster.  After this update, the *Sample* tab shows with one or more ConfigMap ConsoleYAMLSamples present. (link:https://issues.redhat.com/browse/OCPBUGS-41492[*OCPBUGS-41492*])

* Previously, the *Events* page resource type filter incorrectly reported the number of resources when three or more resources were selected.  With this update, the filter always reports the correct number of resources. (link:https://issues.redhat.com/browse/OCPBUGS-38701[*OCPBUGS-38701*])

* Previously, the version number text in the updates graph on the *Cluster Settings* page appeared as black text on a dark background while viewing the page using Firefox in dark mode.  With this update, the text appears as white text. (link:https://issues.redhat.com/browse/OCPBUGS-37988[*OCPBUGS-37988*])

* Previously, *Alerting* pages did not show resource information in their empty state. With this update, resource information is available on the *Alerting* pages. (link:https://issues.redhat.com/browse/OCPBUGS-36921[*OCPBUGS-36921*])

> * Previously, the OLM CSV annotation contained unexpected JSON , which was successfully parsed, but then threw a runtime error when attempting to use the resulting value. With this update, JSON values from OLM annotations are validated before use, and errors will be logged, but the console will not fail when unexpected JSON is received in an annotation. (link:https://issues.redhat.com/browse/OCPBUGS-35744[*OCPBUGS-35744*])

Reviewer: ```suggestion
* Previously, the Operator Lifecycle Manager (OLM) CSV annotation contained unexpected JSON, which was successfully parsed, but then threw a runtime error when attempting to use the resulting value. With this update, JSON values from OLM annotations are validated before use, errors are logged, and the console does not fail when unexpected JSON is received in an annotation. (link:https://issues.redhat.com/browse/OCPBUGS-35744[*OCPBUGS-35744*])
```

> * Previously, silenced alerts were visible on the *Overview* page of the {product-title} web console. This occured because the alerts did not include any external labels. With this release, silenced alerts include the external labels so they can be filtered out so they are not viewable. (link:https://issues.redhat.com/browse/OCPBUGS-31367[*OCPBUGS-31367*])

Reviewer: ```suggestion
* Previously, silenced alerts were visible on the *Overview* page of the {product-title} web console. This occurred because the alerts did not include any external labels. With this release, silenced alerts include the external labels so they are filtered out and are not viewable. (link:https://issues.redhat.com/browse/OCPBUGS-31367[*OCPBUGS-31367*])
```
